core/domain/user_jobs_one_off.py

```diff
@@ -175,6 +175,24 @@ def reduce(key, stringified_values):
                 subscription_services.subscribe_to_collection(key, item['id'])
 
 
+class DashboardStatsOneOffJob(jobs.BaseMapReduceJobManager):
+    """One-off job for populating weekly dashboard stats for all registered
+    users who have a non-None value of UserStatsModel.
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [user_models.UserSettingsModel]
+
+    @staticmethod
+    def map(item):
+        if isinstance(item, user_models.UserSettingsModel):
```

Comment:
I had added this seeing the other occurrence similar to this .Removed this and the other same check as well.